cmd/tsbs_load_timescaledb/main.go

```diff
@@ -98,7 +98,9 @@ func init() {
 type benchmark struct{}
 
 func (b *benchmark) GetPointDecoder(br *bufio.Reader) load.PointDecoder {
-	return &decoder{scanner: bufio.NewScanner(br)}
+	return &decoder{
```

Comment:
These changes seem unnecessary. Let's revert to keep the scope of this PR smaller.

Reply:
Ok, agreed (and this is one-liner case)

Reply:
Summary: updated